versions/3.1.0.md

```diff
@@ -2294,46 +2299,50 @@ The following properties are taken directly from the JSON Schema definition and
-- type - Value MUST be a string. Multiple types via an array are not supported.
-- allOf - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
-- oneOf - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
-- anyOf - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
-- not - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
-- items - Value MUST be an object and not an array. Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema. `items` MUST be present if the `type` is `array`.
-- properties - Property definitions MUST be a [Schema Object](#schemaObject) and not a standard JSON Schema (inline or referenced).
-- additionalProperties - Value can be boolean or object. Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema. Consistent with JSON Schema, `additionalProperties` defaults to `true`.
+- propertyNames
+- contains
+- const
+- examples
+- if / then / else
+- dependentRequired / dependentSchemas
+- deprecated
+- additionalItems
+- additionalProperties
+- unevaluatedItems
+- unevaluatedProperties
+- maxContains
+- minContains
+- readOnly
+- writeOnly
+
+The following properties are taken from the JSON Schema definition but their definitions were adjusted to the OpenAPI Specification.
+
+- exclusiveMinimum - Value can be a numeric value just like JSON Schema, or for backwards compatibility with older drafts it can accept a boolean to act as a modifier to the `minimum` keyword. The numeric value usage is recommended.
+- exclusiveMaximum - Value can be a numeric value just like JSON Schema, or for backwards compatibility with older drafts it can accept a boolean to act as a modifier to the `minimum` keyword. The numeric value usage is recommended.
 - description - [CommonMark syntax](http://spec.commonmark.org/) MAY be used for rich text representation.
 - format - See [Data Type Formats](#dataTypeFormat) for further details. While relying on JSON Schema's defined formats, the OAS offers a few additional predefined formats.
-- default - The default value represents what would be assumed by the consumer of the input as the value of the schema if one is not provided. Unlike JSON Schema, the value MUST conform to the defined type for the Schema Object defined at the same level. For example, if `type` is `string`, then `default` can be `"foo"` but cannot be `1`.
+- default - The value MUST conform to the defined type for the Schema Object defined at the same level. For example, if `type` is `string`, then `default` can be `"foo"` but cannot be `1`. This differs from JSON Schema which only RECOMMENDS the default conforms to the schema.
```

Comment:
From what I have experienced, this is going to break a lot of OAS documents. I'm not saying it is not the right thing to do but we need to plan for it.  I don't see how this can be a 3.1 change.

Reply:
There’s nothing changing here is there? It’s explaining a difference to JSON Schema.

Reply:
@philsturgeon why did you change this at all?  The old wording seems fine, and in particular the "what would be assumed by...." wording correctly captures the way `default` is described.  In particular, it does **not** mean "write this value into the instance if it is missing," which is why that wording is there.

Reply:
I'll revert this. The original seemed confusing so I had a swing at making it more readable, but its possible I just didn't detect the meaning.

Reply:
To add to the confusion, I mixed this up with the requirement that example has to conform with the JSON Schema.  My original comment is invalid!   If I remember correctly, the "what would be assumed" phrase was to discourage tooling from injecting the "default" value and therefore API implementations losing the visibility into whether the consumer provided the value or not.